assets/queries/openAPI/implicit_flow_oauth2/query.rego

```diff
@@ -0,0 +1,22 @@
+package Cx
+
+import data.generic.openapi as openapi_lib
+
+CxPolicy[result] {
+	doc := input.document[i]
+	openapi_lib.check_openapi(doc) != "undefined"
+
+	object.get(doc, "components", "undefined") != "undefined"
+	object.get(doc.components, "securitySchemes", "undefined") != "undefined"
```

Comment:
These lines are unnecessary. 'doc.components.securitySchemes[key]' already check that 'components' and 'securitySchemes' exist